packages/charts/test/reattach.test.js

```diff
@@ -186,6 +186,20 @@ describe('reattach', () => {
     expect(chart.configuration.options.plotOptions.series.stacking).to.be.equal('percent');
   });
 
+  it('should restore configuration done through `updateConfigaration` after reattach', async () => {
+    const wrapper = fixtureSync(`<div><vaadin-chart></vaadin-chart></div>`);
+    const chart = wrapper.firstElementChild;
+    chart.updateConfiguration({ title: { text: 'New title' }, series: [{ name: 'series', data: [1, 2, 3] }] });
+    await nextFrame();
```

Comment:
I wanted to create an empty chart, whose configuration would only be taken by whatever is passed by the `updateConfiguration` method. The chart used by other tests has some `<vaadin-chart-series>`, so that's why I chose to make a new one.